sk-usbhid.c

```diff
@@ -1004,9 +1022,9 @@ sk_sign(uint32_t alg, const uint8_t *data, size_t datalen,
 		skdebug(__func__, "fido_assert_new failed");
 		goto out;
 	}
-	if ((r = fido_assert_set_clientdata_hash(assert, message,
```

Comment:
please add #ifdef WINDOWS block, add a reason for the deviation.

Reply:
Addressed in 6bd15af424.